src/script/team/TeamService.ts

```diff
@@ -87,6 +87,9 @@ export class TeamService {
         },
         [FEATURE_KEY.CONFERENCE_CALLING]: {
           status: FeatureStatus.ENABLED,
+          config: {
+            useSFTForOneToOneCalls: false,
```

Comment:
This is the change that was needed after the Core update

Reply:
I think setting it to false in this PR is the correct call, this only affects the release branch at the moment